ui/.editorconfig

```diff
@@ -0,0 +1,27 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
yes, it contains meaningful information - such as the indent increment size (2) and style (soft) - to help a other developers be consistent